src/Notifier.php

```diff
@@ -59,8 +63,10 @@ public function __construct($opt = [])
             });
         }
 
-        $handler = (isset($this->opt['httpClient']) ? $this->opt['httpClient'] : null);
```

Comment:
This functionality is replaced by using `null` as the default when merging in user-provided options, above.